auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SaveSmartLock.java

```diff
@@ -169,9 +168,7 @@ private void finish() {
     public void saveCredentialsOrFinish(FirebaseUser firebaseUser,
                                         @Nullable String password,
                                         @Nullable IdpResponse response) {
-        if (!mHelper.getFlowParams().smartLockEnabled
-                || !PlayServicesHelper.getInstance(getContext()).isPlayServicesAvailable()
```

Comment:
@samtstern Is there a reason for keeping this here? Since `KickOffActivity` acts as the gatekeeper, shouldn't it always be available?